### PR TITLE
Fix #1810 LongPress works correctly for selecting multiple photos in …

### LINKED ISSUE #1810: OnLongPress issue in favourites activity.
**Actual Behaviour**

When you open an album, after selecting two photos by long pressing on them , all the photos between them are selected. But this doesn't happen in favourites activity. This feature is available for AllPhotosMode.

**Expected Behaviour**

The feature should be present for favourites activity too.

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by angmas1**
Fixed #1810 

Changes: Files changes: LFMainActivity.java, MediaAdapter.java. Fixed bug where LongPressing one photo after selecting one photo in FavouritesActivity didn't automatically select all photos in between.

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/gallery/adapters/MediaAdapter.java [hunk 1]
```diff
@@ -50,6 +50,11 @@ public void updatePlaceholder(Context context) {
         placeholder = (BitmapDrawable) drawable;
     }
 
+    public ArrayList<Media> getList()
+    {
```
Comment (yashk2000) [inline app/src/main/java/org/fossasia/phimpme/gallery/adapters/MediaAdapter.java]: @angmas1 move the opening bracket of the method next to the method declaration as it is in all other methods.

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java [hunk 2]
```diff
@@ -405,6 +409,32 @@ public void selectAllPhotosUpTo(int targetIndex, MediaAdapter adapter) {
         toolbar.setTitle(selectedMedias.size() + "/" + size);
     }
 
+    public void selectAllPhotosUpToFav(int targetIndex)
+    {
```
Comment (yashk2000) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: @angmas1 move the opening bracket of the method next to the method declaration as it is in all other methods.

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java [hunk 3]
```diff
@@ -289,6 +289,10 @@ public boolean onLongClick(View v) {
                     mediaAdapter.notifyItemChanged(toggleSelectPhoto(m));
                     editMode = true;
                 }
+                else
```
Comment (yashk2000) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: @angmas1 move the else up, next to the closing bracket of the if block. Also, there is no need for the braces as your else statement contains only a single line. Make your if-else block similar to the block in lines 277-280.

**Comment by yashk2000** [comment]
@angmas1 please add screenshots of the change also while sending a pull request.

**Comment by angmas1** [comment]
ezgif com-video-to-gif

File: app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java [hunk 4]
```diff
@@ -289,6 +289,9 @@ public boolean onLongClick(View v) {
                     mediaAdapter.notifyItemChanged(toggleSelectPhoto(m));
                     editMode = true;
                 }
+                else {
```
Comment (yashk2000) [inline app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java]: @angmas1 just make one more change here too. Move the else line to the line above it, where the if block ends and remove the braces like in line 280.

**Comment by yashk2000** [comment]
LGTM

**Pull request merged**